core-bundle/src/EventListener/RequestTokenListener.php

```diff
@@ -83,8 +83,8 @@ public function __invoke(RequestEvent $event): void
             'POST' !== $request->getRealMethod()
             || $request->isXmlHttpRequest()
             || false === $request->attributes->get('_token_check')
-            || (!$request->attributes->has('_token_check') && !$this->scopeMatcher->isContaoRequest($request))
             || $this->csrfTokenManager->canSkipTokenValidation($request, $this->csrfCookiePrefix.$this->csrfTokenName)
+            || (!$request->attributes->has('_token_check') && !$this->scopeMatcher->isContaoRequest($request))
```

Comment:
Isn't the `isContaoRequest` call cheaper than `canSkipTokenValidation?

Reply:
I guess the code inspector only counts the number of function calls. Not sure though.

Reply:
I find this version now harder to read as the two `_token_check` conditions are seperated by a line that checks something different.

Also the performance could be worse now for every non-Contao request I think. (But probably doesn’t really matter).